LibraBFT/Impl/Handle/Properties.agda

```diff
@@ -94,6 +89,18 @@ invariantsCorrect pid pre@._ (step-s{pre = pre'} preach (step-peer (step-honest
   postulate -- TODO-3: prove (waiting on: `handle`)
     TODO : RoundManagerInv (msgPool pre) (peerStates pre pid)
 
+qcVoteSigsSentB4
+  : ∀ pid (st : SystemState) {v qc vs pk}
+    → ReachableSystemState st
+    → qc QCProps.∈RoundManager (peerStates st pid)
+    → WithVerSig pk v
+    → vs ∈ qcVotes qc → rebuildVote qc vs ≈Vote v
+    → ¬ (∈GenInfo-impl genesisInfo (proj₂ vs))
+    → MsgWithSig∈ pk (proj₂ vs) (msgPool st)
+qcVoteSigsSentB4 pid st rss qc∈rm sig vs∈qcvs ≈v ¬gen = qcsigsSentB4 qc∈rm sig vs∈qcvs ≈v
```

Comment:
Just noting for the record that this works because we have not yet addressed the issue with `SigForVote∈Rm-SentB4` and signatures in `GenInfo`.